etl/pad/storage/egg_machines_monsters.py

```diff
@@ -0,0 +1,21 @@
+from pad.db.sql_item import SimpleSqlItem
+
+
+class EggMachinesMonster(SimpleSqlItem):
+    "Monsters that appear in each egg_machine"
+    TABLE = 'egg_machines_monsters'
+    KEY_COL = 'emm_id'
+
+    def __init__(self,
+                 emm_id: int = None,
+                 monster_id: int = None,
+                 roll_chance: float = None,
+                 egg_machine_id: int = None):
+        self.emm_id = emm_id
```

Comment:
Possibly rename to egg_machine_monster_id

Reply:
done, also edited the SQL code in the description